distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogValidation.java

```diff
@@ -266,312 +239,96 @@ public void testQueryByWarningFailedValidators() {
   @Test
   public void testFilterPluginWarningsOnly() throws Exception {
-    ingestXmlFromResourceAndWait(WARNING_METACARD);
-    ingestXmlFromResourceAndWait(CLEAN_METACARD);
-    ingestXmlFromResourceAndWait(ERROR_METACARD);
-
-    // Configure invalid filtering
-    configureMetacardValidityFilterPlugin(
-        Collections.singletonList("invalid-state=not-any-users-roles"), getAdminConfig());
+    String warningId = ingestXmlFromResourceAndWait(WARNING_METACARD);
+    String cleanId = ingestXmlFromResourceAndWait(CLEAN_METACARD);
+    String errorId = ingestXmlFromResourceAndWait(ERROR_METACARD);
 
     // Configure to filter metacards with validation warnings but not validation errors
-    configureFilterInvalidMetacards("false", "true", getAdminConfig());
+    Dictionary<String, Object> filterPluginProps =
+        configureMetacardValidityFilterPlugin(
+            Collections.singletonList("invalid-state=data-manager"), false, true, getAdminConfig());
 
     testWithRetry(
         () -> {
-          String query =
-              new CswQueryBuilder().addAttributeFilter(PROPERTY_IS_LIKE, "AnyText", "*").getQuery();
-          ValidatableResponse response =
-              given()
-                  .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_XML)
-                  .body(query)
-                  .post(CSW_PATH.getUrl())
-                  .then();
-
-          // clean metacard should be in results but not invalid one
-          response.body(not(containsString("warning metacard")));
-          response.body(containsString("clean metacard"));
-          response.body(containsString("error metacard"));
+          query(warningId, TRANSFORMER_XML, HttpStatus.SC_NOT_FOUND);
+          query(cleanId, TRANSFORMER_XML, HttpStatus.SC_OK);
+          query(errorId, TRANSFORMER_XML, HttpStatus.SC_OK);
         });
+
+    // Reset plugin
+    configureMetacardValidityFilterPlugin(filterPluginProps, getAdminConfig());
   }
 
   @Test
   public void testFilterPluginErrorsOnly() throws Exception {
-    ingestXmlFromResourceAndWait(ERROR_METACARD);
-    ingestXmlFromResourceAndWait(CLEAN_METACARD);
-    ingestXmlFromResourceAndWait(WARNING_METACARD);
+    String warningId = ingestXmlFromResourceAndWait(WARNING_METACARD);
+    String cleanId = ingestXmlFromResourceAndWait(CLEAN_METACARD);
+    String errorId = ingestXmlFromResourceAndWait(ERROR_METACARD);
 
-    configureMetacardValidityFilterPlugin(
-        Collections.singletonList("invalid-state=not-any-users-roles"), getAdminConfig());
-    configureFilterInvalidMetacards("true", "false", getAdminConfig());
+    // Configure to filter metacards with validation errors but not validation warnings
+    Dictionary<String, Object> filterPluginProps =
+        configureMetacardValidityFilterPlugin(
+            Collections.singletonList("invalid-state=data-manager"), true, false, getAdminConfig());
 
     testWithRetry(
```

Comment:
❓ Do we need this test with retry since we wait for the configuration to be updated in ConfigureTestCommons now?

Reply:
Both the ingest and configuration have waits on them now, so I think they'd be safe to remove, actually